package.json

```diff
@@ -30,10 +30,8 @@
     "access": "public"
   },
   "scripts": {
-    "build": "npm run build:res && snowpack build",
-    "build:res": "rescript build",
-    "clean": "npm run clean:res",
-    "clean:res": "rescript clean",
+    "build": "snowpack build",
```

Comment:
have an issue I founded, when run `build` command.  `index.bs.js` not found
![image](https://user-images.githubusercontent.com/16365952/115139059-14b77400-a05a-11eb-9982-22c065596305.png)

![image](https://user-images.githubusercontent.com/16365952/115139061-1e40dc00-a05a-11eb-8056-b17cd19d738d.png)

Reply:
Hi @ri7nz , thanks for the comment!

Sorry, this PR and the plug-in are still working in progress, it is a experimental works and this might not be merged if things does not go well.

Reply:
Hi @jihchi , thanks for your response. 

I know for this situation, I'm just want to help your work    🙌